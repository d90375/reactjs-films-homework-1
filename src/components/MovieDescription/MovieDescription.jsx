import React from 'react';
import PropTypes from 'prop-types';

import styles from './MovieDescription.scss';

const MovieDescription = ({ description }) => (
  <p className={styles.description}>{ description }</p>
);

MovieDescription.propTypes = {
  description: PropTypes.string.isRequired,
};

export default MovieDescription;
