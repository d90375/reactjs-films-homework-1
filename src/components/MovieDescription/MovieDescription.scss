.description {
  box-sizing: border-box;
  width: 235px;
  padding: 14px;
  text-align: justify;
  color: #fff;
  background-color: rgba(0, 0, 0, 0.6);
}
