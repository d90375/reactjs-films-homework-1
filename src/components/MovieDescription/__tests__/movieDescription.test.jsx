import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MovieDescription from '../MovieDescription';

const description = 'some description';

test('MovieDescription renders correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<MovieDescription description={description} />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
