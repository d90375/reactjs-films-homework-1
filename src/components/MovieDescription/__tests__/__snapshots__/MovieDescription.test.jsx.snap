// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDescription renders correctly 1`] = `
<p
  className="description"
>
  description
</p>
`;
